src/gravity/Artwork.ts

```diff
@@ -1,103 +1,221 @@
+      original_height: R.Number.Or(R.Null),
+      image_url: R.Null.Or(R.String),
+      image_versions: R.Array(R.String),
+      image_urls: R.Record({
+        four_thirds: R.String.Or(R.Undefined),
+        large: R.String.Or(R.Undefined),
+        square: R.String.Or(R.Undefined),
+        tall: R.String.Or(R.Undefined),
+      }),
+      target_supply: R.Boolean,
+    })
+  ),
+  partner: R.Record({
+    partner_categories: R.Array(
+      R.Record({
+        _id: R.String,
+        id: R.String,
+        category_type: R.String,
+        name: R.String,
+        internal: R.Boolean,
+      })
+    ),
+    _id: R.String,
+    id: R.String,
+    default_profile_id: R.String,
+    default_profile_public: R.Boolean,
+    sortable_id: R.String,
+    type: R.String,
+    name: R.String,
+    short_name: R.Null.Or(R.String),
+    pre_qualify: R.Boolean,
+    website: R.String,
+    has_full_profile: R.Boolean,
+    has_fair_partnership: R.Boolean,
+    has_limited_fair_partnership: R.Boolean,
+    profile_layout: R.String,
+    display_works_section: R.Boolean,
+    profile_banner_display: R.Null.Or(R.String),
+    profile_artists_layout: R.Null.Or(R.String),
+    display_artists_section: R.Boolean,
+  }),
+  images: R.Array(
+    R.Record({
+      id: R.String,
+      position: R.Number,
+      aspect_ratio: R.Number.Or(R.Null),
+      downloadable: R.Boolean,
+      original_width: R.Number.Or(R.Null),
+      original_height: R.Number.Or(R.Null),
+      is_default: R.Boolean,
+      image_url: R.String.Or(R.Null),
+      image_versions: R.Array(R.String),
+      image_urls: R.Record({
+        square: R.String.Or(R.Undefined),
+        large_rectangle: R.String.Or(R.Undefined),
+        small: R.String.Or(R.Undefined),
+        tall: R.String.Or(R.Undefined),
+        medium_rectangle: R.String.Or(R.Undefined),
+        large: R.String.Or(R.Undefined),
+        medium: R.String.Or(R.Undefined),
+        larger: R.String.Or(R.Undefined),
+        normalized: R.String.Or(R.Undefined),
+      }),
+      tile_size: R.Number.Or(R.Null),
+      tile_overlap: R.Number.Or(R.Null),
+      tile_format: R.String.Or(R.Null),
+      tile_base_url: R.String.Or(R.Null),
+      max_tiled_height: R.Number.Or(R.Null),
+      max_tiled_width: R.Number.Or(R.Null),
+      gemini_token: R.String.Or(R.Null),
+      gemini_token_updated_at: R.String.Or(R.Null),
+    })
+  ),
+  edition_sets: R.Array(
+    R.Record({
+      id: R.String,
+      size_score: R.Number.Or(R.Null),
+      forsale: R.Boolean,
+      sold: R.Boolean,
+      price: R.String,
+      price_cents: R.Array(R.Number).Or(R.Null),
+      acquireable: R.Boolean,
+      offerable: R.Boolean,
+      dimensions: R.Record({
+        in: R.String.Or(R.Null),
+        cm: R.String.Or(R.Null),
+      }),
+      editions: R.String,
+      display_price_currency: R.String,
+      availability: R.String,
+      availability_hidden: R.Boolean,
+    })
+  ),
+  cultural_makers: R.Array(R.Unknown),
```

Comment:
I'm taking this to mean that there are no artworks with cultural makers published on staging.